src/coreclr/src/hosts/CMakeLists.txt

```diff
@@ -12,3 +12,5 @@ else(CLR_CMAKE_HOST_WIN32)
     add_subdirectory(osxbundlerun)
   endif(CLR_CMAKE_HOST_DARWIN)
 endif(CLR_CMAKE_HOST_WIN32)
+
+add_definitions(-D_FILE_OFFSET_BITS=64)
```

Comment:
This is not correct as this way, it would not have any effect. It needs to be placed right before the `add_subdirectory(unixcoreruncommon)` so that it applies to the directories that get added after it.